tests/clearpart_test.py

```diff
@@ -1,274 +1,240 @@
+""" Clearpart command related unit tests
+"""
 import unittest
 import mock
 
 import blivet
 import blivet.devices
 from blivet.devices.lib import Tags
-from pykickstart.constants import CLEARPART_TYPE_ALL, CLEARPART_TYPE_LINUX, CLEARPART_TYPE_NONE, CLEARPART_TYPE_LIST
-from parted import PARTITION_NORMAL
 from blivet.flags import flags
+from pykickstart.constants import CLEARPART_TYPE_ALL, CLEARPART_TYPE_LINUX
+from pykickstart.constants import CLEARPART_TYPE_NONE, CLEARPART_TYPE_LIST
+from parted import PARTITION_NORMAL
 
 DEVICE_CLASSES = [
     blivet.devices.DiskDevice,
     blivet.devices.PartitionDevice
 ]
 
+# access to protected attributes is needed for setting up the test environment
+# pragma pylint: disable=protected-access
```

Comment:
This warning is disable globally in `runpylint.py` so it shouldn't needed to disable it here -- https://github.com/storaged-project/blivet/blob/2.1-devel/tests/pylint/runpylint.py#L28